### ISSUE TITLE
Support for CurseForge & Modrinth?

### ISSUE THREAD

**Issue by Lylythii** [cyyynthia/pronoundb.org#73]
CurseForge.com uses Twitch.tv accounts to log-in. As such every Curse account/username is associated with a Twitch account/username.
I figure, since Twitch is already supported by PronounDB, perhaps CurseForge could be supported too?

Ideally beside the username on Comments, & on the Members pages. Potentially on the Members list card for mod project pages.

Similarly, Modrinth.com depends on GitHub accounts. Not sure how possible support is, though I believe the Modrinth dev team would be co-operative in implementing this.

Thanks!

Concept:
![image](https://user-images.githubusercontent.com/80999812/201427945-2226a804-ef68-43f0-836f-c7b4f4e5a237.png)

**Comment by cyyynthia** [comment]
Regarding CurseForge, the problem is that while usernames are linked to Twitch account, PronounDB relies on your user ID rather than your user name to identify accounts. This means the client must first convert the username to a Twitch User ID before it can be used to fetch pronouns from PronounDB. That's quite a burden and will break as soon as CurseForge adds new authentications methods other than via Twitch.

Regarding Modrinth, they seem to expose the GitHub user ID internally, so I can harvest it and query the PDB API for pronouns. I don't have experience digging up the internals of a Vue/Nuxt app, but I saw the IDs laying around so it's just a matter of automating the process of finding them :p

tldr I'm unsure how feasible (if at all) integrating Curse is, Modrinth is definitely possible

**Comment by Lylythii** [comment]
I had figured the eventual new authentication system for CurseForge would become a problem, but worth suggesting regardless.
Probably best to revisit later then! If my sources are right the site revamp should be relatively soon.

At the very least Modrinth support would be very much appreciated! :D
